test/song.e2e-spec.ts

```diff
@@ -122,4 +122,30 @@ describe('Song e2e', () => {
 
     expect(songs.items.length).toBeGreaterThanOrEqual(numSongs);
   });
+
+  it('should throw validation error if updating with empty id', async () => {
+    const newName = faker.company.companyName();
+    await expect(
+      app.graphql.mutate(
+        gql`
+          mutation updateSong($input: UpdateSongInput!) {
+            updateSong(input: $input) {
+              song {
+                ...song
+              }
+            }
+          }
+          ${fragments.song}
+        `,
+        {
+          input: {
+            song: {
+              id: '',
+              name: newName,
+            },
+          },
+        }
+      )
+    ).rejects.toThrowError();
```

Comment:
A not found error was previously thrown. So this would've been green before your change. We should at least assert that it's a validation error (has `extensions.codes[0] === 'Validation'`). I'm not sure exactly the syntax of the top of my head. 

It would probably be good to have an `expectValidationError` helper.

Alternatively if you just removed this test I would look the other way haha 😅

Reply:
Oh, you're right.
Seems we disabled the validation pipe in the e2e test app, so it will not go into the validation decorator anymore... lol
https://github.com/SeedCompany/cord-api-v3/blob/master/test/utility/create-app.ts#L33

Reply:
Ah geez. That explains a lot. I wonder if that can be re-enabled now without breaking all the tests.
I'll make a ticket for that. Let's just remove the test here for now since it isn't doing anything.

Reply:
https://github.com/SeedCompany/cord-api-v3/issues/1107